google/cloud/aiplatform_v1beta1/types/model_monitoring.py

```diff
@@ -98,7 +98,7 @@ class TrainingPredictionSkewDetectionConfig(proto.Message):
             skew_thresholds (Sequence[google.cloud.aiplatform_v1beta1.types.ModelMonitoringObjectiveConfig.TrainingPredictionSkewDetectionConfig.SkewThresholdsEntry]):
                 Key is the feature name and value is the
                 threshold. If a feature needs to be monitored
-                for skew, a value threshold must be configed for
+                for skew, a value threshold must be confinged for
```

Comment:
I think they meant to say config'd, which is short for `configured`.

Reply:
change to ```config'd``` or ```configured``` sir ??

Reply:
> change to `config'd` or `configured` sir ??

i think it's be better to be ```configured```

Reply:
configured sounds good 👍

Reply:
okay thanks @dandhlee for comment's and review